notebook/converted/benchmark_results.py

```diff
@@ -1215,6 +1215,57 @@ def return_unlabeled():
 plot
 
 
+# ## Imputation pretraining
+
+# In[3]:
+
+
+in_files = glob.glob('../../results/tissue_impute.*.tsv')
+print(in_files[:5])
+
+
+# In[7]:
+
+
+tissue_metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('tissue_impute.')[-1]
+    model_info = model_info.split('_')
+        
+    supervised_model = '_'.join(model_info[:2])
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-1]
+        
+    tissue_metrics = pd.concat([tissue_metrics, new_df])
+    
+tissue_metrics['train_count'] = tissue_metrics['train sample count']
+tissue_metrics
+
+
+# In[11]:
+
+
+plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='factor(supervised)')) 
+plot += geom_smooth()
+plot += geom_point(alpha=.2)
+plot += ggtitle('Effects of Imputation on Multiclass Tissue Prediction')
+plot += facet_grid('impute_samples ~ .')
+plot
+
+
+# In[5]:
+
+
+plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='factor(impute_samples)')) 
```

Comment:
What is `factor(impute_samples)` here? My guess is that it's no imputation with `impute_samples == 0` vs. imputation with `impute_samples == 28436` - if that's right, it might be good to edit the legend to say that.